testing/workflows/components/workflows.libsonnet

```diff
@@ -166,17 +194,30 @@
                 ],
               },
             },  // checkout
-            $.parts(namespace, name).e2e(prow_env, bucket).buildTemplate("test-deploy", [
+            $.parts(namespace, name).e2e(prow_env, bucket).buildTemplate("setup", [
               "python",
               "-m",
               "testing.test_deploy",
               "--project=mlkube-testing",
```

Comment:
use the local variable project and cluster?

Reply:
Done.